scheduler/src/cook/mesos/unscheduled.clj

```diff
@@ -0,0 +1,143 @@
+;;
+;; Copyright (c) Two Sigma Open Source, LLC
+;;
+;; Licensed under the Apache License, Version 2.0 (the "License");
+;; you may not use this file except in compliance with the License.
+;; You may obtain a copy of the License at
+;;
+;;  http://www.apache.org/licenses/LICENSE-2.0
+;;
+;; Unless required by applicable law or agreed to in writing, software
+;; distributed under the License is distributed on an "AS IS" BASIS,
+;; WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+;; See the License for the specific language governing permissions and
+;; limitations under the License.
+;;
+(ns cook.mesos.unscheduled
+  (:require [clojure.tools.logging :as log]
+            [datomic.api :as d :refer (q)]
+            [cook.mesos.scheduler :as scheduler]
+            [cook.mesos.quota :as quota]
+            [cook.mesos.share :as share]
+            [cook.mesos.util :as util]
+            [clojure.edn :as edn])
+  (import java.util.Date))
+
+(defn check-exhausted-retries
```

Comment:
Use [`job-ent->all-attempts-consumed?`](https://github.com/twosigma/Cook/blob/127d0da245687d67feaf1d627d67a445f7aa4475/scheduler/src/cook/mesos/util.clj#L121)  function to be consistent.